tutorial/05-Animation/assets/sprites/SmallEnemy/index.json

```diff
@@ -47,6 +47,8 @@
       "frameTo": "die-3"
     }
   },
+  "triggerMask":2,
+  "triggerLayer":1,
```

Comment:
Is it possible to configure `Bullet` & `SmallEnemy` into the same layer, so we can omit the configuration for `triggerMask`?

Reply:
Yes, you can do this. For example, you can set the triggerLayer & triggerMask of all sprites to 1, so that all sprites will collide with each other. However, this will cause two problems:

1. Unexpected behavior: bullets will collide with each other.
2. Performance loss.

Reply:
> you can set the triggerLayer & triggerMask of all sprites to 1, so that all sprites will collide with each other.

If the performance is acceptable, I'll suggest we do configuration like this here. That makes the game's behavior closer to it within the previous spx version, and it is simpler not to leverage layer / mask feature.

Reply:
ok